tests/ContainerTest.php

```diff
@@ -95,7 +95,7 @@ public function test_container_build()
         $container->removeEnv();
 
         $this->assertFalse($container->isReady());
-        $this->assertEquals('nginx:1.4', $container->getImage());
+        $this->assertStringEndsWith('nginx:1.23', $container->getImage());
```

Comment:
The change to assertStringEndsWith is useful to ignore the qualified repository bit (regardless of which one it was).

Even in the unqualified case, I could see in [KIND](https://kind.sigs.k8s.io/) that nginx:1.4 would come back with an assertion fault because it got filled in with docker.io/library/nginx:1.4 when queried back out.